tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -1936,4 +1936,15 @@ public function testBug12051(): void
 		$this->analyse([__DIR__ . '/data/bug-12051.php'], []);
 	}
 
+	public function testBug12317(): void
+	{
+		if (PHP_VERSION_ID < 80000) {
+			$this->markTestSkipped('Test requires PHP 8.0.');
+		}
+
+		$this->checkExplicitMixed = true;
+		$this->checkImplicitMixed = true;
+		$this->analyse([__DIR__ . '/data/bug-12317.php'], []);
```

Comment:
Would be a good idea to add an example that actually produces an error, to verify the args still look like they should.

Reply:
Thank you for your suggestion. I added examples and fixed a bug caught by them.